docs/section10/10-14.md

```diff
@@ -0,0 +1,142 @@
+        - SYSTEM TYPE 9  
+        - Check else if the system is system type 9: `elif baseline_system_type_compare(hvac_sys_type_b, HVAC_SYS.SYS_9, true):`  
+            - Loop through each of the efficiency_metric_types associated with the heating system until COMBUSTION_EFFICIENCY is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`
+                    - Check if the efficiency metric equals COMBUSTION_EFFICIENCY: `if heating_system_b.efficiency_metric_types[x] == "COMBUSTION_EFFICIENCY":` 
+                        - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`  
+                    - Else, do nothing and move on to the next metric: `Else:`               
+            - Set expected_baseline_eff_b = 80% Ec:`expected_baseline_eff_b = 0.80`  
+      
+        - SYSTEM TYPE 3 and 3a  (<225,000)
+        - Check else if the system type is 3 or 3a:`elif any(baseline_system_type_compare(hvac_sys_type_b, target_sys_type, true) for target_system_type in [HVAC_SYS.SYS_3,HVAC_SYS.SYS_3a]):`  
+            - Get dictionary list of baseline zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_b = get_hvac_zone_list_w_area (B_RMR)`  
+            - Get list of zones that the HVAC system serves (this is to get the zone associated with sys 3 or 4): `zone_list_b = hvac_zone_list_w_area_dict_b[hvac_b.id]["ZONE_LIST"]`  
+            Get the heating capacity:
+            - Check if rated heating capacity does not equal null, if it does not equal null then set the heating capacity equal to this data element's value: `if heating_system_b.rated_capacity != Null:`
+                - Set heating capacity equal to the rated heating capacity: `total_heat_capacity_b = heating_system_b.rated_capacity`
+            - Else if, check that design heating capacity does not equal null:`Elif heating_system_b.design_capacity != Null:`  
+                - Set heating capacity equal to the design heating capacity: `total_heat_capacity_b = heating_system_b.design_capacity`
+            - Else, set the was_heat_capacity_not_defined boolean variable equal to TRUE: `Else: was_heat_capacity_not_defined = TRUE`
+            - Check if capacity is <225,000 Btuh (below checks if the modeled value is equal to the most stringent size category which is <225,000 for when capacity is not defined) or (if the aggregation factor was defined (i.e., does not equal Null) and the resultant capacity is less than 225000) or if the aggregation factor = Null : `if total_heat_capacity_b < 225000 OR was_heat_capacity_not_defined boolean variable == TRUE OR (zone_list_b[0].aggregation_factor != Null AND total_heating_capacity_b/zone_list_b[0].aggregation_factor < 225000) OR zone_list_b[0].aggregation_factor == Null:`  
```

Comment:
This made me realized that we should be producing an UNDETERMINED outcome when the aggregation factor is not defined AND when it is needed (sometimes it not needed to figure out the expected efficiency). I revised the code accordingly. It is not defaulted to one. Thanks for the suggestion but due to moving this along I left the code as is since it accomplishes what we need it to (please check that it does though!).

Reply:
> just so I'm clear on the intent of how zone.aggregation_factor is used - looks like you are assuming that if it does not exist, it is essentially equal to 1. Is this correct?
> 
> the math / logic in this rule could be simplified by creating a dictionary that mapped a number for zone aggregation factor to a number (either 1 for null entries, or the number, if it exists). This would look like this:
> 
> ```
> for zone in zone_list
>    zone_aggregation_dict = {}
>    if zone.aggregation_factor == Null
>       zone_aggregation_dict[zone] = 1
>    else
>       zone_aggregation_dict[zone] = zone.aggregation_factor
> ```
> 
> & would be done at the start of the rule
> 
> the above line is then simplified to:
> 
> `if (total_heat_capacity_b/zone_aggregation_dict[zone_list_b[0].aggregation_factor]) < 225000 OR was_heat_capacity_not_defined boolean variable == TRUE OR zone_list_b[0].aggregation_factor == Null:`
> 
> similar simplifications could be made throughout the rule

@KarenWGard I had the same idea of defaulting the aggregation factor to 1 in the implementation of this rule!